src/common/utils/formatter/formatter.ts

```diff
@@ -75,26 +51,56 @@ export function getMiddleNumber(values: number[]): number {
   }
 }
 
+type FormatType = "a" | "b";
+
+const scales: Record<FormatType, Record<string, number>> = {
+  a: {
+    k: 1e3,
+    m: 1e6,
+    b: 1e9,
+    t: 1e12
+  },
+  b: {
```

Comment:
is it possible to perform some recursion here ? it would be much easier to read when MiB = 1024 * KiB and so on ...

Reply:
Yeah, at least get those values out of this file into some constants.